.deepsource.toml

```diff
@@ -0,0 +1,64 @@
+version = 1
```

Comment:
What kind of file is this? What is this doing?

Reply:
Sorry I forgot to delete that before the merge. It's AI helping with bugs, performance, documentation issues, anti-patterns, security issues, coverage issues and so on. There are over 1000+ issues. You should try it yourself and see how it can help. I just wanted to quickly look trough polkadot since I like the project and think it's one of the most promising projects in crypto 2023. Best Regards Philip

Reply:
are you planning to add more things to this pr and clean that up or is this it

Reply:
I just wanted to help a little now when I have free time over. Happy to help, hope you also can adress others on the issue I posted regarding the CVE. Best regards the gitlabs team. 
![polkadot](https://user-images.githubusercontent.com/87861683/224757509-f8594f27-214a-49bc-835b-d63a3ee622e5.jpg)